### PR TITLE
yazi: 0.1.4 -> 0.1.5

### PR DESCRIPTION
**Pull request by realeinherjar**
## Description of changes

https://github.com/sxyazi/yazi/releases/tag/v0.1.5

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

Cc @XYenon (thanks for the work maintaining :))

### CODE REVIEW

**Review by XYenon — CHANGES_REQUESTED** [review]
Yazi 0.1.5 added some shell completions, you can add them to nix like https://github.com/XYenon/nur-packages/commit/2c201c14ded01af4f7325d5cdaf14b7ffe9f232d

**Comment by realeinherjar** [comment]
> Yazi 0.1.5 added some shell completions, you can add them to nix like [XYenon/nur-packages@2c201c1](https://github.com/XYenon/nur-packages/commit/2c201c14ded01af4f7325d5cdaf14b7ffe9f232d)

Thanks! I've added them, and amended to the commit.

**Review by XYenon — APPROVED** [review]

**Comment by XYenon** [comment]
@ofborg eval

**Review by matthiasbeyer — APPROVED** [review]

File: pkgs/applications/file-managers/yazi/default.nix [hunk 1]
```diff
@@ -60,6 +61,10 @@ rustPlatform.buildRustPackage rec {
     ''
       wrapProgram $out/bin/yazi \
          --prefix PATH : "${makeBinPath runtimePaths}"
+      installShellCompletion --cmd ${pname} \
```
Comment (eclairevoyant) [inline pkgs/applications/file-managers/yazi/default.nix:64]:
```suggestion
      installShellCompletion --cmd yazi \
```

**Review by eclairevoyant — CHANGES_REQUESTED** [review]
Other than that, builds and runs.

Result of `nixpkgs-review pr 262500` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages blacklisted:</summary>
  <ul>
    <li>nixos-install-tools</li>
    <li>tests.nixos-functions.nixos-test</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>yazi</li>
  </ul>
</details>

File: pkgs/applications/file-managers/yazi/default.nix (hunk 1 above)
Comment (realeinherjar) [reply pkgs/applications/file-managers/yazi/default.nix:64]: If we add this contribution, we need to change also https://github.com/XYenon/nur-packages/commit/2c201c14ded01af4f7325d5cdaf14b7ffe9f232d @XYenon.

Comment (XYenon) [reply pkgs/applications/file-managers/yazi/default.nix:64]: > If we add this contribution, we need to change also [XYenon/nur-packages@2c201c1](https://github.com/XYenon/nur-packages/commit/2c201c14ded01af4f7325d5cdaf14b7ffe9f232d) @XYenon.

Yeah, I have updated my nur

Comment (teto) [reply pkgs/applications/file-managers/yazi/default.nix:64]: is the nur needed now there is a flake \o/ ?

**Review by teto — APPROVED** [review]
Yahoo, a new release of this amazing software \o/

**Review by eclairevoyant — APPROVED** [review]

**Comment by tukanoidd** [comment]
when on unstable? (no rush ofc, im just curious, cuz i never figured out how frequent unstable releases are)

**Comment by eclairevoyant** [comment]
Releases are continuous
see https://nixpk.gs/pr-tracker.html?pr=262500 for this PR specifically

**Comment by tukanoidd** [comment]
> Releases are continuous see [nixpk.gs/pr-tracker.html?pr=262500](https://nixpk.gs/pr-tracker.html?pr=262500) for this PR specifically

Oh, thanks!

**Comment by teto** [comment]
one great way to avoid the wait is to checkout the nixos-unstable branch and cherry-pick whatever PR you are interested . Then you `nixos-rebuild -f ~/nixpkgs`

**Comment by tukanoidd** [comment]
> one great way to avoid the wait is to checkout the nixos-unstable branch and cherry-pick whatever PR you are interested . Then you `nixos-rebuild -f ~/nixpkgs`

I'm using flakes to configure my system, not sure how i would do it with that setup. But I also just found out that yazi has a flake as well, so gonna be using that for the forseeable future

**Pull request merged**